### PR TITLE
BUG: Prevent warnings from scipy.stats

### PR DESCRIPTION
**Pull request by Dimchord**
This prevents the scipy runtime warnings when calling anova_lm

### CODE REVIEW

File: statsmodels/stats/anova.py [hunk 1]
```diff
@@ -333,19 +331,9 @@ def anova_lm(*args, **kwargs):
         raise ValueError("Multiple models only supported for type I. "
                          "Got type %s" % str(typ))
 
-    ### COMPUTE Anova TYPE I ###
-
-    # if given a single model
-    if len(args) == 1:
-        return anova_single(*args, **kwargs)
```
Comment (josef-pkt) [inline statsmodels/stats/anova.py]: why did you remove this?

File: statsmodels/stats/anova.py [hunk 2]
```diff
@@ -132,16 +132,14 @@ def anova1_lm_single(model, endog, exog, nobs, design_info, table, n_rows, test,
...
-    table.loc['Residual', ['sum_sq','df', test, pr_test]] = (model.ssr,
-                                                             model.df_resid,
-                                                             np.nan, np.nan)
+    table.loc['Residual', ['sum_sq','df']] = model.ssr, model.df_resid
+    if test == 'F':
+        table[test] = ((table['sum_sq'] / table['df']) /
+                       (model.ssr / model.df_resid))
+        table[pr_test] = stats.f.sf(table["F"], table["df"],
+                                    model.df_resid)
+        table.loc['Residual', [test, pr_test]] = np.nan, np.nan
```
Comment (josef-pkt) [inline statsmodels/stats/anova.py]: those changes look fine AFAICS

**Comment by josef-pkt** [comment]
AFAICS, test coverage looks good for this, so it should have could any errors.
appveyor failure is unrelated, it's the occasional `test_simulate.py` hang

File: statsmodels/stats/anova.py (hunk 1 above)
Comment (Dimchord) [reply statsmodels/stats/anova.py]: That is duplicate code, also a couple of lines above the diff.

Comment (josef-pkt) [reply statsmodels/stats/anova.py]: I see now, looks good

**Comment by josef-pkt** [comment]
I'm merging this

@Dimchord Thank you
